code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -382,7 +382,20 @@
 		name = "small carton"
 		desc = "A small carton, intended for holding drinks."
 
-
+/obj/item/reagent_containers/food/drinks/honeycomb
+	name = "Honeycomb"
+	desc = "A honeycomb made by an apid. It seems to be overflowing with honey."
```

Comment:
Get a better desc: comment about how much stuff in it isn't very recommended; or you have to manually update the desc when this reagent container gets empty.

Reply:
done